responses/matchers.py

```diff
@@ -152,17 +166,22 @@ def match(request: PreparedRequest) -> Tuple[bool, str]:
         reason = ""
         request_params = request.params  # type: ignore[attr-defined]
         request_params_dict = request_params or {}
-        valid = (
-            params is None
-            if request_params is None
-            else params_dict == request_params_dict
-        )
+
+        if not strict_match:
+            # filter down to just the params specified in the matcher
+            request_params_dict = {
+                k: v for k, v in request_params_dict.items() if k in params_dict
+            }
+
+        valid = sorted(params_dict.items()) == sorted(request_params_dict.items())
 
         if not valid:
             reason = "Parameters do not match. {} doesn't match {}".format(
                 _create_key_val_str(request_params_dict),
                 _create_key_val_str(params_dict),
             )
+            if not strict_match:
+                reason += "\nNote that you use not strict parameters check."
```

Comment:
```suggestion
                reason += "\nYou can use `strict_match=False` to use enable matches."
```

What do you think of this? It makes the error more instructive on what a next step could be.

Reply:
agree with the idea, but I think the message should be `"You can use 'strict_match=True' to do a strict parameters check."`
since now this reason is triggered when `strict_match=False`